Source/Peripheral.swift

```diff
@@ -161,19 +167,23 @@ public class Peripheral {
 	 - Returns: Observable that emits `Next` with array of `Service` instances, once they're discovered.
 	 Immediately after that `.Complete` is emitted.
 	 */
-	public func discoverIncludedServices(includedServiceUUIDs: [CBUUID]?,
-		forService service: Service) -> Observable<[Service]> {
+	public func discoverIncludedServices(includedServiceUUIDs: [CBUUID]?, forService service: Service) -> Observable<[Service]> {
```

Comment:
This function is very similar to `discoverServices`. Maybe it's possible to extract common code for them?